### PR TITLE
Add info on ODH deployment modification.

### PR DESCRIPTION
**Pull request by martinpovolny**
review here: https://github.com/martinpovolny/continuous-deployment/blob/next_steps/docs/modify-odh-deployment.md

### CODE REVIEW

File: docs/modify-odh-deployment.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Modify your ODH deployment
+
+Below are the steps to modify your ODH deployment. Add comonents, content or anything else.
```
Comment (anishasthana) [inline docs/modify-odh-deployment.md]: `Add comonents, content or anything else.` -> ``Add comment, content, or anything else.`

File: docs/modify-odh-deployment.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Modify your ODH deployment
```
Comment (anishasthana) [inline docs/modify-odh-deployment.md:1]: What are we trying to modify here exactly? ODH itself? Or apps that being deployed?

Comment (martinpovolny) [reply docs/modify-odh-deployment.md:1]: We are going to modify the deployment, meaning the components that are deployed.

File: docs/modify-odh-deployment.md (hunk 1 above)
Comment (martinpovolny) [reply docs/modify-odh-deployment.md]: "components" -- ODH components. One letter missing.

**Review by anishasthana — APPROVED** [review]
lgtm

**Comment by HumairAK** [comment]
can you fix the pre-commit errors, thanks! :)

edit: rebasing should be sufficient

**Review by tumido — APPROVED** [review]

**Comment by hemajv** [comment]
instructions worked for me :+1:

**Pull request merged**